codespell_lib/tests/test_basic.py

```diff
@@ -53,22 +83,21 @@ def test_basic(tmpdir, capsys):
         f.write('tim\ngonna\n')
     assert cs.main(fname) == 2, 'with a name'
     assert cs.main('--builtin', 'clear,rare,names,informal', fname) == 4
-    capsys.readouterr()
-    assert cs.main(fname, '--builtin', 'foo') == 1  # bad type sys.exit(1)
-    stdout = capsys.readouterr()[1]
-    assert 'Unknown builtin dictionary' in stdout
+    code, _, stderr = cs.main(fname, '--builtin', 'foo', std=True)
+    assert code == EX_USAGE  # bad type
+    assert 'Unknown builtin dictionary' in stderr
     d = str(tmpdir)
-    assert cs.main(fname, '-D', op.join(d, 'foo')) == 1  # bad dict
-    stdout = capsys.readouterr()[1]
-    assert 'cannot find dictionary' in stdout
+    code, _, stderr = cs.main(fname, '-D', op.join(d, 'foo'), std=True)
+    assert code == EX_USAGE  # bad dict
+    assert 'cannot find dictionary' in stderr
     os.remove(fname)
 
     with open(op.join(d, 'bad.txt'), 'w') as f:
         f.write('abandonned\nAbandonned\nABANDONNED\nAbAnDoNnEd')
     assert cs.main(d) == 4
-    capsys.readouterr()
-    assert cs.main('-w', d) == 0
-    assert 'FIXED:' in capsys.readouterr()[1]
+    code, _, stderr = cs.main('-w', d, std=True)
+    assert code == 0
```

Comment:
EX_OK

Reply:
I don't see value in updating all of these tests, I think using `0` interchangeably with `EX_OK` is fine

Reply:
Okay.